vignettes/MsExperiment.Rmd

```diff
@@ -244,12 +244,250 @@ We can access the raw data as long as the mzML files that were used to
 generate it still exist in their original location, which is the case
 here as they were saved in the `rpx` cache directory.
 
-```{r}
+```{r, eval = FALSE}
 sp <- spectra(msexp)
 sp
 plotSpectra(sp[1000])
 ```
 
+
+# Linking experimental data to samples
+
+For some experiments and data analyses an explicit link between data, data files
+and respective samples is required. Such links enable an easy (and error-free)
+subsetting or re-ordering of a whole experiment by sample and would also
+simplify coloring and labeling of the data depending on the sample or of its
+variables or conditions.
+
+Below we generate an `MsExperiment` object for a simple experiment consisting of
+a single sample measured in two different injections to the same LC-MS setup.
+
+```{r}
+mse <- MsExperiment()
```

Comment:
May be call this one `mse2` or something else to avoid any confusion with the one above.

Reply:
renamed it to `lcmse` (LC-MS experiment).